nixos/modules/virtualisation/libvirtd.nix

```
@@ -160,7 +160,7 @@ in {
       etc."qemu/bridge.conf".text = lib.concatMapStringsSep "\n" (e:
         "allow ${e}") cfg.allowedBridges;
       systemPackages = with pkgs; [ libressl.nc iptables cfg.package cfg.qemuPackage ];
-      etc.ethertypes.source = "${pkgs.iptables}/etc/ethertypes";
+      etc.ethertypes.source = "${pkgs.iptables-nftables-compat}/etc/ethertypes";
```

```suggestion
      etc.ethertypes.source = "${pkgs.ebtables}/etc/ethertypes";
```

Yup, somewhat expected, as this is L2 territory, so we could also use the ebtables version.